package/MDAnalysis/analysis/base.py

```diff
@@ -281,26 +298,51 @@ def run(self, start=None, stop=None, step=None, verbose=None):
             stop frame of analysis
         step : int, optional
             number of frames to skip between each analysed frame
+        frames : array_like, optional
+            array of integers or booleans to slice trajectory
         verbose : bool, optional
             Turn on verbosity
+
+
+    Example
+    -------
+       import MDAnalysis as mda
+       from MDAnalysisTests.datafiles import TPR, XTC
+       from MDAnalysis.analysis import rms
+
+       u = mda.Universe(TPR, XTC)
+
+       rmsd = rms.RMSD(u, select='name CA')
+       # A list of frames to run the analysis on can be passed
```

Comment:
Could you also throw in an example using start/stop/step? That way we have every use case here?

Reply:
done